app/packages/core/src/components/Actions/ActionsRow.tsx

```diff
@@ -162,13 +162,14 @@ const Tag = ({
   const [available, setAvailable] = useState(true);
   const labels = useRecoilValue(fos.selectedLabelIds);
   const samples = useRecoilValue(fos.selectedSamples);
-  const readOnly = useRecoilValue(fos.readOnly);
+  const canTag = useRecoilValue(fos.canTagSamplesOrLabels);
+  const disableTag = canTag.enabled !== true;
```

Comment:
```suggestion
  const disableTag = !canTag.enabled;
```
assuming enabled is a required property and not being set means should not be enabled